src/tests/welcome.test.js

```diff
@@ -0,0 +1,21 @@
+const chai = require('chai');

```

Comment:
Could you use ES6 syntax when you are writing tests?

Reply:
> Could you use ES6 syntax when you are writing tests?

may be setting up ES6 babel could be another task apart from this one am working on

Reply:
I think there is no story of setting up `ES6 babel` so I think we can use that syntax in whatever we are doing because after merging all of the branches into develop branch, your `ES5` will become an error because there will be `eslint`

Reply:
I think there is no story of setting up `ES6 babel` so I think we can use that syntax in whatever we are doing because after merging all of the branches into develop branch, your `ES5` will become an error because there will be `eslint`

Reply:
and another thing that I found which can help you is that error is coming from the line `41 - 46` if you delete those lines, there won't be any error anymore